packages/ckeditor5-ckbox/docs/features/ckbox.md

```diff
@@ -236,7 +236,7 @@ Also, make sure to include the translation file after loading the CKBox library:
 
 ```html
 <script src="https://cdn.ckbox.io/ckbox/latest/ckbox.js"></script>
-<script src="https://cdn.ckbox.io/CKBox/1.2.1/translations/es.js"></script>
+<script src="https://cdn.ckbox.io/ckbox/latest/translations/es.js"></script>
```

Comment:
Using `/latest` is a bit risky as, at some point, something may stop working due to breaking changes that are not visible using the `latest` version.

If we want to update it, I would add a note/warning infobox about it.

Reply:
In commit.